Loops/sixTask.js

```diff
@@ -0,0 +1,5 @@
+let value;
+
+do {
+  value = prompt("Введите число, большее 100?", 0);
+} while (value <= 100);
```

Comment:
Please check the solution on learnjs for this task. Looks like you missed part of verification.

Reply:
Yes. I fixed it.